examples/loss_comparison_2.py

```diff
@@ -9,15 +9,18 @@
 import pathlib
 import numpy as np
 import pandas as pd
-from joblib import Memory
+from datetime import datetime
 import matplotlib.pyplot as plt
 
+from joblib import Memory, Parallel, delayed
+
 
 from carpet.utils import init_vuz
 from carpet.datasets import synthetic_1d_dataset
-from carpet import ListaTV, LpgdTautString, CoupledIstaLASSO
-from carpet.iterative_solver import IstaAnalysis, IstaSynthesis
+from carpet.metrics import compute_prox_tv_errors
 from carpet.loss_gradient import analysis_primal_obj, tv_reg
+from carpet import ListaTV, LpgdTautString, CoupledIstaLASSO  # noqa: F401
+from carpet.iterative_solver import IstaAnalysis, IstaSynthesis
 
 
 OUTPUT_DIR = pathlib.Path('outputs_plots')
```

Comment:
Pass it as cmd-line argument?

Reply:
This is hard coded everywhere else so let's not make the script more complex for now.